packages/plugins/plugin-pipeline/src/components/PipelineSettings.tsx

```
@@ -138,35 +138,35 @@ export const ProjectObjectSettings = ({ classNames, project }: ProjectObjectSett
-      <List.Root<Project.Column>
-        items={project.columns}
-        isItem={Schema.is(Project.Column)}
+      <List.Root<Pipeline.Column>
+        items={pipeline.columns}
+        isItem={Schema.is(Pipeline.Column)}
         getId={(column) => column.view.dxn.toString()}
         onMove={handleMove}
       >
         {({ items: columns }) => (
           <>
             <div role='list' className={mx(listGrid)}>
```

Biome (2.3.14)

[error] The elements with this role can be changed to the following elements:
<ol>
<ul>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)